include/stream-client/connector/impl/connection_pool.ipp

```diff
@@ -1,12 +1,18 @@
 #pragma once
 
+#include "stream-client/logger.hpp"
+
 namespace stream_client {
+
+using namespace logging;
```

Comment:
`using namespace` in headers is prohibited. Such declarations spread in every file where this header will be included and it is bad. Info [1](http://www.gotw.ca/gotw/053.htm), [2](https://abseil.io/tips/119), [3](https://abseil.io/tips/130)

Reply:
Yep, I forget about it. Thank you for your attention and links with detailed information.